access/slackbot/main.go

```diff
@@ -47,14 +47,14 @@ type requestData struct {
 	roles []string
 }
 
-type slackData struct {
+type SlackData struct {
```

Comment:
this probably doesn't need to be exported

Reply:
The linter complained because it wasn't exported, figured it was easier just to export it. I can just revert it and add a comment for the nolint.

Reply:
It seems that it doesn't really matter is anything exported or not since it's an executable binary. So eventually, I rewrote everything to be private.
However, in some cases I had to make some types exported to use with `text/template`. I was confused and started to make everything public in new plugins but forgot to fix it in this place.